2011-03-05  Jonathan Nieder  <jrnieder@gmail.com>

	* sysdeps/unix/sysv/linux/sys/param.h: Fix an #ifndef __undef_ARG_MAX
	where #ifdef was intended.  The intent is to prevent ARG_MAX from
	being defined by the kernel headers.

---
 sysdeps/unix/sysv/linux/sys/param.h |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

--- a/sysdeps/unix/sysv/linux/sys/param.h
+++ b/sysdeps/unix/sysv/linux/sys/param.h
@@ -1,4 +1,5 @@
-/* Copyright (C) 1995-1997,2000,2001,2003,2008 Free Software Foundation, Inc.
+/* Copyright (C) 1995-1997,2000,2001,2003,2008,2011
+   Free Software Foundation, Inc.
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
@@ -28,7 +29,7 @@
 #include <linux/param.h>
 
 /* The kernel headers defines ARG_MAX.  The value is wrong, though.  */
-#ifndef __undef_ARG_MAX
+#ifdef __undef_ARG_MAX
 # undef ARG_MAX
 # undef __undef_ARG_MAX
 #endif
